streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -1035,10 +1093,10 @@ private Stream<Task> streamOfNonPausedTasks() {
     private class StateUpdaterMetrics {
         private static final String STATE_LEVEL_GROUP = "stream-state-updater-metrics";
 
-        private static final String IDLE_RATIO_DESCRIPTION = RATIO_DESCRIPTION + "being idle";
-        private static final String RESTORE_RATIO_DESCRIPTION = RATIO_DESCRIPTION + "restoring active tasks";
-        private static final String UPDATE_RATIO_DESCRIPTION = RATIO_DESCRIPTION + "updating standby tasks";
-        private static final String CHECKPOINT_RATIO_DESCRIPTION = RATIO_DESCRIPTION + "checkpointing tasks restored progress";
+        private static final String IDLE_RATIO_DESCRIPTION = WINDOWED_RATIO_DESCRIPTION_PREFIX + "of the time this thread spend being idle";
+        private static final String RESTORE_RATIO_DESCRIPTION = WINDOWED_RATIO_DESCRIPTION_PREFIX + "of the time this thread spend restoring active tasks";
```

Comment:
```suggestion
        private static final String RESTORE_RATIO_DESCRIPTION = WINDOWED_RATIO_DESCRIPTION_PREFIX + " restoring active tasks";
```

not sure about this one

spent


<img width="338" height="107" alt="Image" src="https://github.com/user-attachments/assets/c474a5e7-0503-4d4e-8f94-3e3ebec7b222" />


spend

<img width="786" height="808" alt="Image" src="https://github.com/user-attachments/assets/3e2b19ac-c198-4e66-a038-347d49f225a1" />


idk if this is any good, all I know this string occurs 22 times making it kind of dry.

"of the time this thread spen"



<img width="272" height="31" alt="Image" src="https://github.com/user-attachments/assets/c92a58fa-45c0-4291-b3c3-7227fd790336" />